templates/emails/reviewers_group_invitation.rst

```diff
@@ -0,0 +1,18 @@
+Hello,
+
+We hope you are in good health and doing well.
+This email is in regards to PyCon India 2020 reviewer group. Pycon India is going virtual this year considering the current situations. We (conference organizers) take this as an opportunity to put more efforts in making this conference a great experience for attendees in other ways.
+
+We would like to put some more effort in bringing up some amazing talks to be presented this  Pycon 2020. We are creating a team of volunteers to review the talk proposals that should be presented in Pycon. Since this is a peer review process and driven by volunteers, we are looking for more volunteers to join us and help in making this a success. We believe you have great experience in Python and/or Python community and we would love if you can be a part of this group.
+
+Your role will be mainly to review the content of the talks submitted. You will be volunteering for a few talks you see fit as per your skills (we will help you with this) and provide your feedback if this should be a part of Pycon 2020. 
```

Comment:
```suggestion
Your role will be mainly to review th talk proposals submitted. You will be volunteering for a few talks you see fit as per your skills (we will help you with this) and provide feedback if those talks could be the right fit for Pycon 2020. 
```

I don't get what this means, `You will be volunteering for a few talks you see fit as per your skills (we will help you with this)`

Reply:
```suggestion
Your role will be mainly to review the talks submitted in your area of expertise. We would like you to get your feedback to find a fantastic talk lineup for PyCon India 2020.
```

The second line can become very confusing. Hence, it is better to refactor.